pkg/driver/node.go

```diff
@@ -60,35 +57,35 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 	}
 
 	context := req.GetVolumeContext()
-	dnsname := context[volumeParamsDnsName]
-	mountname := context[volumeParamsMountName]
-	volumeType := strings.ToLower(context[volumeParamsVolumeType])
+	dnsName := context[volumeContextDnsName]
+	volumePath := context[volumeContextVolumePath]
```

Comment:
what's the distinction between mount name and volume path

Reply:
I'm not extremely familiar with the Lustre use case of mount name, but from my understanding they are essentially the same. Volume path is the mount path of a given OpenZFS volume. For example this could be fsx/volume1 or fsx/volume2.

Mount name isn't a term used by OpenZFS, thus why I changed it to volume path.

Reply:
Yup that makes sense, I assume you tested with the name change?

Reply:
Yea, I was able to mount and write to the volume after the name change.